tests/API/XCCDF/default_cpe/cpe-dict.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<cpe-list xmlns="http://cpe.mitre.org/dictionary/2.0"
+          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+          xsi:schemaLocation="http://cpe.mitre.org/dictionary/2.0 http://cpe.mitre.org/files/cpe-dictionary_2.3.xsd">
+      <cpe-item name="<<CPE>>">
+            <title xml:lang="en-us">Applicable example platform</title>
+            <check system="http://oval.mitre.org/XMLSchema/oval-definitions-5" href="cpe-oval.xml">oval:x:def:1</check>
+      </cpe-item>
```

Comment:
Hmm, do you think there should be one more cpe item with the `cpe:/o:example:not_applicable:5`?

Reply:
No, don't think so. It won't be applicable if it does not exists (as a CPE entity anywhere), so why bother?

Reply:
Well, my idea was to test the scenario, where you have a content including a CPE dictionary for a certain product (`cpe:/o:example:not_applicable:5` in this case), but this product is not matched as the scanned system has a different CPE.